plugins/optimization-detective/detect.js

```diff
@@ -239,6 +224,18 @@ export default async function detect( {
 		log( 'Proceeding with detection' );
 	}
 
+	/** @type {Extension[]} */
+	const extensions = [];
+	for ( const extensionModuleUrl of extensionModuleUrls ) {
+		const extension = await import( extensionModuleUrl );
+		extensions.push( extension );
+		// TODO: There should to be a way to pass additional args into the module. Perhaps extensionModuleUrls should be a mapping of URLs to args. It's important to pass webVitalsLibrarySrc to the extension so that onLCP, onCLS, or onINP can be obtained.
```

Comment:
we also have JavaScript hooks you could leverage if the idea is to make this extensible.

Reply:
These aren't available as script modules though, right?